Watchman.Discord/Areas/Protection/Controllers/MuteUserController.cs

```diff
@@ -65,6 +68,9 @@ public async Task MutedUsers(MutedUsersCommand mutedUsersCommand, Contexts conte
                 return;
             }
             await this._directMessagesService.TrySendEmbedMessage(contexts.User.Id, mutedUsersMessageData.Title, mutedUsersMessageData.Description, mutedUsersMessageData.Values);
+            var text = "Wysłano wiadomość z listą wyciszonych użytkowników!";
```

Comment:
to możesz przenieść do responses
w embed jeszcze tego nie obsługujemy, ale w normalnych wiadomościach lepiej przez responses

Reply:
2. nie dotyczące tego fragmentu, tylko ogólnie
popraw -mutedusers tak aby nie było wywoływania query dla każdego muteEvent (zobacz sobie w logach po wywołaniu tej komendy ile razy wywoływane jest Query)
najlepiej jest pobrać wszystkie NotUnmutedMuteEvents dla danego serwera, potem użytkowników i po tych użytkownikach iterować sprawdzając czy ma on jakiś muteEvent